src/client/view/mostUsedWords.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import { SlidingSearchBar } from './searchBar';
+
+const WordInfo = ({ wordInfo }) => (
+  <tr>
+    <td className='right aligned column'>{wordInfo.count}</td>
+    <td className='left aligned column'>{wordInfo.word}</td>
+    <td className='right aligned column'>
+      <div className='ui checkbox'>
+        <input type='checkbox'  defaultChecked='true'/>
+      </div>
+    </td>
+  </tr>
+);
+
+class Words extends Component {
+  componentDidMount() {
+    $('.ui.checkbox').checkbox();
+  }
+
+  componentDidUpdate() {
+    $('.ui.checkbox').checkbox();
+  }
+
+  render() {
+    return (
+      <div style={{ 'height': '300px', 'overflow-y': 'scroll' }}>
+        <table className='ui very basic celled table'>
+          <tbody>
+          { this.props.words.map((wordInfo) => <WordInfo wordInfo={wordInfo}></WordInfo>) }

```

Comment:
Change WordInfo component to WordItem so it reads <WordItem wordInfo={wordInfo}></WordItem>